pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -860,7 +868,7 @@
     "tradfri" = ps: with ps; [ ]; # missing inputs: pytradfri[async]
     "trafikverket_train" = ps: with ps; [ pytrafikverket ];
     "trafikverket_weatherstation" = ps: with ps; [ pytrafikverket ];
-    "transmission" = ps: with ps; [ transmissionrpc ];
+    "transmission" = ps: with ps; [ ]; # missing inputs: transmissionrpc
```

Comment:
Looks like that the package was [updated](https://github.com/NixOS/nixpkgs/tree/nixos-unstable/pkgs/development/python-modules/transmission-rpc) (from 0.11 to 3.2.2) and renamed in #112237.

 I think that without some feedback of transmission user it's hard to tell with the integration will work with 3.2.2.

Reply:
Worse.

```
{
  "domain": "transmission",
  "name": "Transmission",
  "config_flow": true,
  "documentation": "https://www.home-assistant.io/integrations/transmission",
  "requirements": ["transmissionrpc==0.11"],
  "codeowners": ["@engrbm87", "@JPHutchins"]
}
```

They are two different packages.

https://pypi.org/project/transmissionrpc/
https://pypi.org/project/transmission-rpc/

Reply:
I actually had to downgrade 3.2.2 back to 0.11 because it no longer worked.

Reply:
It needs an entry in `PKG_PREFERENCES` of `parse-requirements.py`.

I guess that Martin will run `parse-requirements.py` before finishing this PR anyways and with the new entry in the dict we will end-up a `component-packages.nix` that contains `transmissionrpc` again.

Reply:
I tried exactly that yesterday,

```diff
diff --git a/pkgs/servers/home-assistant/parse-requirements.py b/pkgs/servers/home-assistant/parse-requirements.py
index eeb117a984e..18577e68f36 100755
--- a/pkgs/servers/home-assistant/parse-requirements.py
+++ b/pkgs/servers/home-assistant/parse-requirements.py
@@ -42,6 +42,7 @@ PKG_PREFERENCES = {
     "tensorflow-bin_2": "tensorflow",
     "tensorflowWithoutCuda": "tensorflow",
     "tensorflow-build_2": "tensorflow",
+    "transmissionrpc": "transmission-rpc",
     "whois": "python-whois",
 }
```

but that went nowhere.

Reply:
In #112237 it looked like that `transmission-rpc` is a drop-in replacement for `transmissionrpc`. But the it's a fork which introduces a new module. 

https://github.com/NixOS/nixpkgs/pull/115186 will re-add `transmissionrpc`.

Reply:
Rebased, thank you!